bc_obps/reporting/models/report_person_responsible.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+
+from registration.models.time_stamped_model import TimeStampedModel
+from registration.models.user_and_contact_common_info import UserAndContactCommonInfo
+from reporting.models.report_version import ReportVersion
+
+
+class ReportPersonResponsible(UserAndContactCommonInfo, TimeStampedModel):
+    # A report version can have multiple persons responsible
```

Comment:
Is this comment correct? I thought a report version can only have one person responsible?